Feature/add common covariates

halfhorst opened the pull request:
This adds support for a list of covariates that Abie indicated he would want to use for simulations some time going forward. Because these and the existing covariates are all formatted the same, I made a change to retain the age column as a general rule. All smoke tests pass except Sanofi

collijk commented on src/vivarium_inputs/data_artifact/loaders.py:
```diff
@@ -222,20 +222,26 @@ def get_population_data(_, measure, location, __):
-        data = core.get_covariate_estimates([covariate], [location])
-        if covariate.name == "age_specific_fertility_rate":
-            columns = ["location", "mean_value", "lower_value", "upper_value", "sex_id", "year_id", "age_group_id"]
+        if covariate.name in ["age_specific_fertility_rate", "live_births_by_sex",
```
Instead of explicitly white listing things, I think it might be worthwhile to look for particular data shapes.  Let people pull anything with the columns ["location", "mean_value", "lower_value", "upper_value",  "sex_id", "year_id", "age_group_id"] and then mark things for special consideration.

halfhorst replied:
So you mean pull first and just check what columns are there ?

collijk replied:
Yeah, pull, make sure columns are there and make sure they have values we know how to interpret.  Maybe raise a warning for, e.g., only one sex_id or only a subset of age_group_ids present.

collijk commented on src/vivarium_inputs/data_artifact/loaders.py:
```diff
@@ -222,20 +222,26 @@ def get_population_data(_, measure, location, __):
-        data = core.get_covariate_estimates([covariate], [location])
-        if covariate.name == "age_specific_fertility_rate":
-            columns = ["location", "mean_value", "lower_value", "upper_value", "sex_id", "year_id", "age_group_id"]
+        if covariate.name in ["age_specific_fertility_rate", "live_births_by_sex",
```
Also, does this still work for age_specific fertility rate?.  That should only have values for sex_id 2.

halfhorst commented:
I broke things being more permissive with covariates. Standby

halfhorst commented:
I added back a function call I left out that normalizes column names. Also added support for age_group _22 (which doesn't actually break things, but results in an age column of NaN!). It's ready to look at
